benchmarks/freetype2-2017/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get update &&  \
+    apt-get install -y \
+    make \
+    autoconf \
+    libtool \
+    libarchive-dev \
+    zlib1g-dev
+
+COPY * $SRC/
```

Comment:
Can we make this a common step, why do we need this in every Dockerfile.

Reply:
The OSS-Fuzz dockerfiles already have this line so making it a common step would requiring changing it more.
I think it's bad for two reasons:
1. Less consistency with OSS-Fuzz.
2. More magic that saves developers very little but adds complexity to build code.

Reply:
Ok, fine as-is.